DParser2/Resolver/ExpressionSemantics/Evaluation.IsExpression.cs

```diff
@@ -84,7 +84,8 @@ private bool evalIsExpression_WithAliases(IsExpression isExpression, AbstractTyp
 			if (retTrue)
 			{
 				foreach (var kv in tpl_params)
-					ctxt.CurrentContext.DeducedTemplateParameters[kv.Key] = kv.Value;
+					if (kv.Key != null && kv.Value != null)
```

Comment:
Ich wüsste gern, bei welchem Zustand das null ist -- Test? :O

Reply:
Ja, Tests. Hatte ich schon befürchtet, aber die zusammenzustellen ist natürlich nochmal mehrfacher Aufwand. So weit ich das beurteilen konnte, treten die üblicherweise auf, nachdem schon vorher irgendeine Evaluation schiefgegangen ist.

Reply:
Ich habe mir neulich einmal die Codebase von xfwm und xfce angeschaut, und mit Schrecken feststellen müssen, dass es dafür gar keine Tests gibt :I -- wie gut, dass es welche in D_Parser gibt, die integrativ neue Anpassungen testen :sweat_smile: